pareto/strategic_water_management/strategic_produced_water_optimization.py

```diff
@@ -3402,6 +3488,454 @@ def SeismicActivityExceptionRule(model, k, t):
     return model
 
 
+def operational_hydraulics(model, opt):
```

Comment:
I believe this function will throw an exception if it is called on a model for which `config.hydraulics` is `Hydraulics.false`. I think it would be good to print a specific warning/error message in this telling the user not to call this function on a model with that config option.

Reply:
Doesn't hurt to add a warning message, but the way the model is build now, this method is NOT called if Hydraulics config is false, so I don't thing it is essential. 
See line 6257:
```
if model.config.hydraulics != Hydraulics.false:
        hydraulics_model = operational_hydraulics(model, opt)
```

Reply:
i agree. i would think the warning is probably not necessary

Reply:
I see - `pipeline_hydraulics()` (formerly `operational_hydraulics()`) is not meant to be called directly - rather, it is called within `solve_model()`, and it is only called there `if model.config.hydraulics != Hydraulics.false`.

Resolving.